source/dump.f90

```diff
@@ -186,27 +186,26 @@ subroutine dump_lines(n,i,ix)
 
   use mod_common_track
 
-  implicit none
-
   integer, intent(in) :: n,i,ix
+  real(kind=fPrec) tmpTas(6,6), tmpInv(6,6), tmpClo(6)
 
-  if (ldump(0)) then
+  call dump_getTasMatrix(ix, tmpInv, tmpTas, tmpClo)
+
+  if(ldump(0)) then
     ! Dump at all SINGLE ELEMENTs
-    if (ndumpt(0) == 1 .or. mod(n,ndumpt(0)) == 1) then
+    if(ndumpt(0) == 1 .or. mod(n,ndumpt(0)) == 1) then
       if ((n >= dumpfirst(0)) .and. ((n <= dumplast(0)) .or. (dumplast(0) == -1))) then
-        call dump_beam_population(n, i, ix, dumpunit(0), dumpfmt(0), ldumphighprec, &
-          dump_normMat(dump_nMatMap(ix))%orbit, dump_normMat(dump_nMatMap(ix))%invtas)
+        call dump_beam_population(n, i, ix, dumpunit(0), dumpfmt(0), ldumphighprec, tmpClo, tmpInv)
```

Comment:
Does tmpClo and tmpInv actually make any sense here, or are you just sending them as placeholders (and somewhere else -- possibly just in the manual -- taking care that a normalized dump for ALL is not attempted?)

Reply:
Huh?

I'm using the getTasMatrix routine to get the tmpTas, tmpInv and tmpClo before passing them on. The rotuine checks that they are set first to avoid out of bounds issues. That is the entire point of this fix.